app/turing_machine.rb

```diff
@@ -63,7 +63,7 @@ def parse_symbol(unary_code)
     when 1 then '0'
     when 2 then '1'
     when 3 then '_'
-    else unary_code.length.to_s
+    else (61 + unary_code.length).chr
```

Comment:
ascii code for A: 65

```suggestion
    else (A_CODE_ASCII - OFFSET + unary_code.length).chr
```